addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -165,15 +167,9 @@ function defaultOptions(defaultObject, object) {
     return newObject;
 }
 function getImageFiles(dataTransfer) {
-    let files;
-    if (!dataTransfer.items) {
-        files = [...dataTransfer.items]
-            .filter(item => item.kind === 'file' && item.type.includes('image/'))
-            .map((item) => item.getAsFile());
-    } else {
-        files = [...dataTransfer.files];
-    }
-    return files || [];
+    return [...dataTransfer.items]
+        .filter(item => item.kind === 'file' && item.type.includes('image/'))
+        .map((item) => item.getAsFile());
```

Comment:
It could be possible for `dataTransfer.items` is not defined so we should check `dataTransfer.files` as a fallback.

Reply:
@Goaman 
That should not happen. `files` is the old interface, `items` replaces it:
![image](https://user-images.githubusercontent.com/24205914/199694972-5c2d8981-312a-4923-8a91-81d030b493cf.png)

It used to be an issue because only Chrome had implemented `items` but this seems [not to be the case anymore](https://caniuse.com/?search=datatransfer.items).

If you have a concrete case I'd make the required changes but in the meantime I don't think that's necessary.